lib/vf_settokenvalue.cpp

```diff
@@ -618,7 +618,7 @@ namespace ValueFlow
                         bits = settings.platform.long_bit;
                 }
                 if (bits > 0 && bits < MathLib::bigint_bits)
-                    v.intvalue &= (((MathLib::biguint)1)<<bits) - 1;
+                    v.intvalue &= (1ULL<<bits) - 1;
```

Comment:
this would not work well the day we switch to 128-bit bigints.. however I guess there are plenty of other code that will not work perfectly that day.

Reply:
This actually causes a compilation failure when using the 128-bit int. That's how I came across this.